_data/simple-icons.json

```
@@ -12438,6 +12438,11 @@
             "hex": "F27557",
             "source": "https://quip.com"
         },
+        {
+            "title": "Quizlet",
+            "hex": "4255FF",
+            "source": "https://quizlet.com"
```

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Quizlet_Logo_2021.svg"
```

Since you derived the SVG from Wikipedia Commons, that's where we should link to for the source. See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines.